Mention about default config template path in config doc (#1230)
openshift/microshift: docs/howto_config.md

* Environment variables
* Configuration file

> The MicroShift configuration file must be located at `~/.microshift/config.yaml` (user-specific) and `/etc/microshift/config.yaml` (system-wide), while the former takes precedence if it exists. If you install Microshift RPM, a sample config file present in `/etc/microshift/config.yaml.default` which can be used as a template to create your own configuration file.

Reviewer: ```suggestion
The MicroShift configuration file must be located at `~/.microshift/config.yaml` (user-specific) and `/etc/microshift/config.yaml` (system-wide), while the former takes precedence if it exists. A sample
`/etc/microshift/config.yaml.default` configuration file is installed by the MicroShift RPM and it can be
used as a template when customizing MicroShift.
```
